Exiled.Events/EventArgs/Player/TogglingFlashlightEventArgs.cs

```diff
@@ -31,33 +33,32 @@ public class TogglingFlashlightEventArgs : IPlayerEvent, IDeniableEvent
         /// <param name="newState">
         ///     <inheritdoc cref="NewState" />
         /// </param>
-        /// <param name="isAllowed">
-        ///     <inheritdoc cref="IsAllowed" />
-        /// </param>
-        public TogglingFlashlightEventArgs(Player player, FlashlightItem flashlight, bool newState, bool isAllowed = true)
+        public TogglingFlashlightEventArgs(ReferenceHub hub, FlashlightItem flashlight, bool newState)
         {
-            Player = player;
+            Player = Player.Get(hub);
             Flashlight = (Flashlight)Item.Get(flashlight);
+            initialState = newState;
             NewState = newState;
-            IsAllowed = isAllowed;
         }
 
         /// <summary>
         ///     Gets the <see cref="API.Features.Items.Flashlight" /> being toggled.
         /// </summary>
         public Flashlight Flashlight { get; }
 
-#pragma warning disable SA1623 // Property summary documentation should match accessors
         /// <summary>
-        ///     Gets or sets the new flashlight state.
+        ///     Gets or sets a value indicating whether is will be the NewState of the Light.
```

Comment:
```suggestion
        ///     Gets or sets the new flashlight state.
```

Reply:
I don't see a need to change that, plus the change itself is grammatically wrong.

Reply:
1